authapi/models.py

```diff
@@ -11,12 +11,19 @@ class SeedOrganization(models.Model):
 
 
 class SeedPermission(models.Model):
-    # TODO: Implement SeedPermission model
-    pass
+    permission_type = models.CharField(max_length=128)
+    object_id = models.CharField(max_length=64)
+    namespace = models.CharField(max_length=64)
+    created_at = models.DateTimeField(auto_now_add=True)
+    modified_at = models.DateTimeField(auto_now=True)
+    archived = models.BooleanField(default=False)
 
 
 class SeedTeam(models.Model):
-    # TODO: Implement SeedTeam model
+    name = models.CharField(max_length=64)
```

Comment:
Only noticed this now, but in numi's api, been using `title` instead of `name` for things, since `name` sometimes means something else in numi. It would be helpful if we could use the same naming.